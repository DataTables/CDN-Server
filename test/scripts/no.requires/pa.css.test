pa-1.0.0/fred.css
404
