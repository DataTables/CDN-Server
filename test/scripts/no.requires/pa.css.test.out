extensionsURL >pa-1.0.0<
extensionsList >ProductA 1.0.0<
source >"pa-1.0.0"<
testString >test string<
./ProductA-1.0.0/js/producta.css


